xtable-core/src/test/java/org/apache/xtable/conversion/TestConversionController.java

```diff
@@ -396,6 +522,17 @@ private SyncResult buildSyncResult(SyncMode syncMode, Instant lastSyncedInstant)
         .build();
   }
 
+  private SyncResult buildSyncResult(
+      SyncMode syncMode, Instant lastSyncedInstant, Duration duration) {
+    return SyncResult.builder()
+        .mode(syncMode)
+        .lastInstantSynced(lastSyncedInstant)
+        .syncStartTime(Instant.now().minusSeconds(duration.getSeconds()))
```

Comment:
I am a little worried this can cause some flakiness. Can we pass in the actual start time?